source/index.html.md

```diff
@@ -133,6 +133,35 @@ key = OpenSSL::PKey::RSA.new(key_in_pem_string)
 key.sign(OpenSSL::Digest.new('sha256'), body).unpack('H*').first
 ```
 
+## Accounts and keys
+
+There are two types of accounts:
+
+* A merchant account belongs to a merchant.
+* A partner account belongs to you, i.e. a partner that is tech-wise integrated
+  into the transaction gateway.
+
+Merchant accounts can process transactions whereas partner accounts cannot.
+
+An API key (UUIDv4) points to an account (either a merchant account or a partner
+account). An account can have multiple API keys pointing to it; this allows for
+easy rolling of API keys. A merchant account usually has one API key.
```

Comment:
```suggestion
easy rolling of API keys. A merchant account usually uses one API key.
```

A very minor comment that you may discard at your whim :)

Reply:
ba0261d